docs/running-jobs/job-resources.md

```diff
@@ -35,9 +35,10 @@ These are the partitions available on Alpine.
 | amem<sup>1</sup> | High-memory           | 22          | 48 or 64  |   16          | ~4.0            |  4H,  7D                 |
 | csu       | Nodes contributed by CSU     | 77         | 32 or 48   |   3.74        | ~1.1            | 24H, 24H                
 
-> <sup>1</sup> The `amem` partition is limited to 96 cores across *all running amem jobs.* For example, you can run one 96-core job or up to two 48-core jobs, four 24-core jobs, ninty-six 1-core jobs, etc.  If you need more memory or cores, please contact <rc-help@colorado.edu>.
-> > <sup>2</sup> Approximate billing weight per core on a given resource. This will vary depending on the quantity of CPUs, RAM and (if applicable) GPUs requested.  For example, on an `amem` node, requesting 1 CPU for one hour will result in 4.0 SUs being charged. 
-> 
+> <sup>1</sup> The `amem` partition is limited to 96 cores across *all running amem jobs.* For example, you can run one 96-core job or up to two 48-core jobs, four 24-core jobs, ninty-six 1-core jobs, etc.  If you need more memory or cores, please contact <rc-help@colorado.edu>.  
+
+> <sup>2</sup> Approximate billing weight per core on a given resource. This will vary depending on the quantity of CPUs, RAM and (if applicable) GPUs requested.  For example, on an `amem` node, requesting 1 CPU for one hour will result in 4.0 SUs being charged. 
+
 
 In addition to these partitions, Research Computing also provides specialized partitions for interactive and test jobs, as well as compilation. These partitions allow quick access to a reserved set of cores provided for testing and interactive use. 
 
```

Comment:
The `Partitions` and `Quality of Service` sections on this page contain tables about partitions specific to Alpine. These are basically duplicate tables. I really don't like this as we have to change information in two places and that is hard. I suggest removing these tables and linking to the appropriate tables in `clusters/alpine/alpine-hardware.md`.

Reply:
@SchneiderCode This has not been resolved!

Reply:
Please put link to Alpine QOS like in the previous section